site/content/integrate/slash-commands/outgoing-oauth-connections/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Outgoing OAuth connections
+heading: Outgoing OAuth connections
+weight: 20
+---
+
+As of Mattermost v9.6, you can now integrate Mattermost with custom integrations hosted within your internal OAuth infrastructure. More specifically, integrations like [slash commands](https://developers.mattermost.com/integrate/slash-commands/custom), [outgoing webhooks](https://developers.mattermost.com/integrate/webhooks/outgoing), [interactive dialogs](https://developers.mattermost.com/integrate/plugins/interactive-dialogs), and [interactive messages](https://developers.mattermost.com/integrate/plugins/interactive-messages) now support communicating via both the [Client Credentials](https://oauth.net/2/grant-types/client-credentials) and [Password](https://oauth.net/2/grant-types/password) OAuth 2.0 grant types.
```

Comment:
Note that these are currently implemented:

- Client credentials grant type
- Slash commands
- Outgoing webhooks


While these are not:
- Password grant type (implemented on server but not frontend)
- Interactive dialogs
- Interactive messages

I drafted these this way to detail how the full feature should work, though should probably be removed to agree with the initial release of this feature

Reply:
Great context, @mickmister, thank you! I agree that developer documentation should cover available product functionality. Any features or functionality coming in a future iteration can be documented in parallel to releases.